modules/installation-configuration-parameters.adoc

```diff
@@ -1727,6 +1727,50 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the type of disk.
 |`premium_LRS` or `standardSSD_LRS`. The default is `premium_LRS`.
 
+|`platform.azure.defaultMachinePlatform.encryptionAtHost`
+|Enables host-level encryption for compute machines. You can enable this encryption alongside user-managed server-side encryption. This feature encrypts temporary, ephemeral, cached and un-managed disks on the VM host. This is not a prerequisite for user-managed server-side encryption.
+|`true` or `false`. The default is `false`.
+
+|`platform.azure.defaultMachinePlatform.osDisk`
```

Comment:
I think it doesn't need to mention this, since you already added its parameters under this object, `platform.azure.defaultMachinePlatform.osDisk.diskSizeGB` and `platform.azure.defaultMachinePlatform.osDisk.diskType`